engineering/workflow/default-workflow.md

```diff
@@ -60,29 +60,148 @@ Last day of Iteration
+  - Send a message to the PM at least one day prior to the planning session with a list of issues you feel need to be included in the upcoming sprint.
+  - Please include as much context as needed for the PM to understand why these issues need to be in the upcoming sprint
+  - The PM will take your suggestions under advisement and make a best effort to include them, but cannot guarantee that to be the case as other priorities and capacity will need to be considered. 
+
+#### 🗺️ Roadmap & Initiative Review
+
+The Product Manager will:
+
+- **Review roadmap tab for issues with dates**  
+  - Pull in overdue items.
+  - Reassign unrealistic dates to achievable ones.
+  - Ensure no items lag behind without follow-up.
+ 
+#### ✅ Preparing for Planning
+
+The Team will:
+
+- **Ensure issues are current**  
+  - All team members should update and clean up any issues to which they are assigned prior to the planning session
+  - Ensure the use of the Standard Definition of Done available in the General Issue template where possible.
+
+The Engineering Manager will:
+
+- **Report on last sprint’s progress**  
+  Add totals for:
+  - ✅ Done  
+  - 📂 Open  
+  - ❌ Unplanned  
+  to the previous sprint’s **Status Update**.
+
+#### 📆 At the Planning Session 
+
+The Engineering Manager will:
 
-The goal of this meeting is to synchronize with one another and commit to work to do during the iteration that gets us closer to achieving the initiatives objectives.
+#### 🔁 Sprint Review & Retrospective
+
+- **Review last sprint’s outcomes**
+  - Compare *actual velocity* vs. target
+  - Document the *status update*
+  - Identify questions or blockers on leftover cards
+  - Incorporate *retro feedback* to adjust workflows
+
+#### 🎯 Define Sprint Goals
+
+The Engineering Manager will:
+
+- **Establish sprint themes**
+  - Ask the Head of Product and the Technical Lead to share focus areas or big goals
+  - Invite input from the whole team for broader context
+
+The Team will:
+
+- **Identify a “1” (low-effort card)**  
+  Start with the easiest, quickest win for the sprint.
+
+#### 🃏 Committing Cards
+
+The Team will:
+
+- **Sort Board by date**  
+  - Focus first on items due or overdue before exploring the rest.
+  - Make sure to check dated items in the Up Next, Refined, Upcoming and In Flight columns are not being left behind
+
+- **Start from the top of ‘Up Next’**
+  - ✍️ Summarize each card for shared understanding  
+  - 🧠 Estimate effort using Planning Poker or expert judgment  
+  - 🙋 Assign responsible team member(s)  
+  - 🔖 Move card to **Committed** and tag with sprint iteration
+
+- **Prioritize initiative-linked and Contract Deliverable tasks**  
+  - Reinforce the importance of completing tasks tied to Initiatives in Flight and Contract Deliverables.
+
+- **Ensure coverage of key categories**  
+  Explicitly address:
+  - Contract Deliverables
```

Comment:
Let's turn this into a numbered list, and explicitly state this is the priority order